source/includes/fact-validate-repair-option.rst

```diff
@@ -29,7 +29,8 @@ The repair fixes these issues:
 - If corrupt documents with invalid BSON data are found, the documents
   are removed.
 
-.. seealso:: :option:`--repair <mongod --repair>` option for
-   :binary:`~bin.mongod`
+.. seealso::
```

Comment:
FYI : This was a fix for a see also directive that was showing empty on the page. Cleaned up with this effort.